src/transformer/matcher/matcher.ts

```diff
@@ -18,8 +18,8 @@ export function isFromTsAutoMock(signature: ts.Signature): boolean {
         return false;
     }
 
-    const createMockTs: string = path.join(__dirname, `src/transformer/create-mock.ts`);
-    const createMockListTs: string = path.join(__dirname, `src/transformer/create-mock-list.ts`);
+    const createMockTs: string = path.join(__dirname, `../create-mock.d.ts`);
```

Comment:
Why is this targeting the `.d.ts`?

Reply:
Before I was exporting and publishing also the ts file(create-mock.ts), Now I'm only exporting the d.ts file.